solution/index.js

```diff
@@ -0,0 +1,170 @@
+function handleDBclick (event){
+    if(event.target.tagName !== "LI") return;
+    let task = event.target.innerText;
+    let targetArr = event.target.closest("section").dataset.action;
+    let taskI = tasks[targetArr].indexOf(task);
+    event.target.contentEditable = true;
+    event.target.focus();
+    event.target.onblur = () => {
+        if(event.target.innerText=="") { //prevent saving empty task
+            event.target.focus();
+            return;
+        }
+        let updateTask = event.target.innerText;
+        tasks[targetArr][taskI] = updateTask;
+        localStorage.setItem("tasks", JSON.stringify(tasks)); //save to local storage
+        event.target.contentEditable = false
+    }
+    
+}
+//load tasks from local storage
+function loadTasks () {
+    if(!localStorage.getItem("tasks")) //if no tasks found in local storage save empty tasks jason
+        localStorage.setItem("tasks", JSON.stringify(tasks))
+    tasks = JSON.parse(localStorage.getItem("tasks"));
+    let classI = 0;
+    let newLi;
+    for(let taskArr in tasks){ //loop over task arrays (to-do \ in progress \ done)
+        for(let i = 0; i < tasks[taskArr].length; i++){ //loop over each array
+            newLi =  document.createElement("li");
+            newLi.classList.add("task");
+            newLi.innerHTML = tasks[taskArr][i];
+            document.getElementsByTagName("ul")[classI].append(newLi);
+        }
+        classI++;
+    }
+}
+
+//handle task mouse over
+function handleMouseOver(event) {
+    if(event.target.tagName !== "LI") return; 
+    event.target.classList.add("active");
+    document.addEventListener("keydown", keyPressMove);
+}
+
+//handles task move to another section, on mouse over + alt+1-3
+function keyPressMove(e) {
+    let task = document.getElementsByClassName("active")[0]; //get the active (hovered) task
+    let prevUl = task.closest("section").dataset.action; //get active task section
+    let prevTaskindex = tasks[prevUl].indexOf(task.innerText) //get active task index
+    if(e.altKey && (e.key === "1" || e.key === "2" || e.key === "3")){
+        let indexUl = e.key - 1;
+        let targetUl = document.getElementsByTagName("ul")[indexUl]; //get destination UL
+        targetUl.insertBefore(task, targetUl.childNodes[0]); //insert active task to destination UL
+        if(e.key === "3"){ //if task moved to DONE show the welldone gif
+            document.getElementById("wellDone").style.display = "block";
+            setTimeout(function() {document.getElementById("wellDone").style.display = "none";}, 4000);
+        }
+        tasks[targetUl.parentNode.dataset.action].unshift(task.innerText); //push task first on target array
+        tasks[prevUl].splice(prevTaskindex, 1); //remove task from previous array
+        localStorage.setItem("tasks", JSON.stringify(tasks)); //save to local storage
+
+    }
+}
+
+//on mouse out - task in no longer active and keydown event removed
+function mouseOutElement(event) {
```

Comment:
you added the event to the document level so it will run more times than you need. yes you created an if to handle the problem but thats not the correct way to handle this task.